packages/cdk/lib/stacks/engines/miniwdl-engine-construct.ts

```diff
@@ -93,6 +93,7 @@ export class MiniwdlEngineConstruct extends EngineConstruct {
       jobDefinitionArn: this.miniwdlEngine.headJobDefinition.jobDefinitionArn,
       rootDirS3Uri: rootDirS3Uri,
       vpc: props.contextParameters.usePublicSubnets ? undefined : props.vpc,
+      customEnvs: props.contextParameters.adapterCustomEnvs,
```

Comment:
Use `customWesEnvVars` for this engine and all other engine constructs so that it is more obvious where these variables are being used.